v2/tools/generator/internal/astmodel/known_locals_set.go

```diff
@@ -28,15 +28,44 @@ func NewKnownLocalsSet(idFactory IdentifierFactory) *KnownLocalsSet {
 	}
 }
 
+// flectOverrides is an ordered list of overrides for flect pluralization
+// Earlier overrides take precedence over later ones
+var flectOverrides = []struct {
+	single string
+	plural string
+}{
+	{"FIPS", "FIPS"},
+	{"Id", "Ids"},
+	{"Ip", "Ips"},
+	{"knownAs", "knownAs"},
+	{"ssh", "ssh"}, // workaround flect bug - see https://github.com/gobuffalo/flect/pull/65
+}
+
 // CreateSingularLocal creates a new unique Go local variable for a single value with one of the specified suffixes.
 func (locals *KnownLocalsSet) CreateSingularLocal(nameHint string, suffixes ...string) string {
 	hint := flect.Singularize(nameHint)
+	lowerHint := strings.ToLower(nameHint)
+	for _, o := range flectOverrides {
```

Comment:
minor: could this be turned into a function so that its implementation could be shared?

I'm thinking something like:
```
func (locals *KnownLocalsSet) applyFlectOverrides(flectValue string, originalValue string) string {
	...
}
```

Although maybe better to do what my other comment suggested of a flect-wrapper so we can use it in `type_name` too

Reply:
Agreed. But a job for a different PR.